pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -202,19 +202,11 @@ func (db *DB) GetBandwidthAllocationBySignature(signature []byte) ([][]byte, err
 }
 
 // AddTTLToDB adds TTL into database by id
-func (db *DB) AddTTLToDB(id string, expiration int64) error {
+func (db *DB) AddTTLToDB(id string, expiration, size int64) error {
 	defer db.locked()()
 
 	created := time.Now().Unix()
-	_, err := db.DB.Exec("INSERT or REPLACE INTO ttl (id, created, expires) VALUES (?, ?, ?)", id, created, expiration)
-	return err
-}
-
-// UpdateTTLSize adds stored data size into database by id
-func (db *DB) UpdateTTLSize(id string, size int64) error {
-	defer db.locked()()
-
-	_, err := db.DB.Exec("UPDATE ttl SET size=? WHERE id=?", size, id)
+	_, err := db.DB.Exec("INSERT or REPLACE INTO ttl (id, created, expires, size) VALUES (?, ?, ?, ?)", id, created, expiration, size)
```

Comment:
Only use "insert" here and let it fail when there's a constraint violation.

Reply:
Tests fail because they are running in parallel using the same id